### ISSUE TITLE
Spinner freeze on mobile

### ISSUE THREAD

**Issue by onigetoc** [ressio/lazy-load-xt#95]
Image gif Spinner freeze on mobile.
May be it would be interesting to do a css3 spinner?
http://projects.lukehaas.me/css-loaders/

**Comment by dryabov** [comment]
Yes, I know issue with gif animation on mobile, most likely it's disabled for powersafe reason. In turn, it was a reason to include work with https://github.com/daneden/animate.css into lazyloadxt's docs and demo.
